docs/src/modules/drawerSubNavigation.js

```diff
@@ -114,9 +122,62 @@ class ModulesDrawerSubNavigation extends React.PureComponent {
                 <SubNavigation.Item label="Title Bar" />
                 <SubNavigation.Item label="Wing" />
                 <SubNavigation.Item label="Details" />
+                <SubNavigation.Item label="Sticky" />
             </SubNavigation>
         );
     }
+
+    _onSubNavClick(index) {
```

Comment:
Do we need this function?  It looks like a duplicate of `static onSubNavClick(index)`...?